### PR TITLE
Print out full arrays, not just the first 100 elements

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

**Comment by janedegtiareva** [comment]
Fix for https://github.com/near/near-shell/issues/314

**Comment by janedegtiareva** [comment]
Tested: see screenshot 
Screenshot (19)

**Comment by janedegtiareva** [comment]
Tested on windows with a view function call that returns an array of 150 elements.

File: utils/inspect-response.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
 const util = require('util');
 module.exports = (response) => {
-    return util.inspect(response, { showHidden: false, depth: null, colors: true });
+    return util.inspect(response, { showHidden: true, depth: null, colors: true, maxArrayLength: null });
```
Comment (vgrichina) [inline utils/inspect-response.js:4]: why is `showHidden` needed?

**Comment by janedegtiareva** [comment]
> lgtm, but I think long term we need to expose some controls by user to make stuff more / less verbose

Yup, agreed.

File: utils/inspect-response.js (hunk 1 above)
Comment (janedegtiareva) [reply utils/inspect-response.js:4]: I found it useful since it printed out array length.

**Pull request merged**